File: javascript/change.js
var age = 36;
var names = ['John', 'Jane', 'Doe'];

function changeAge(age) {
//  age = 39;
  return 39;
}

function changeNames(names) {
    names[0] = 'John Doe';
    return names;
    }

age = change(age);
changeNames(names);

console.log(age); 
console.log(names[0]);

File: swift/change.swift
var age: Int = 39
var name: [String] = ["John", "Doe"]

func change_age(age: inout Int) {
    age = 20
}
func change_name(name: inout [String]) {
    name = ["Jane", "Doe"]
}

change_age(age: &age)
change_name(name: &name)

print(age)
print(name)
